servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -175,6 +176,7 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
         systemProperty junit5TimeoutParamName, "$junit5Timeout"
         systemProperty "junit.jupiter.extensions.autodetection.enabled", "true"
 
+        ignoreFailures = true
```

Comment:
Why is this required now?

Reply:
Intellij IDEA has a feature of multiple changelists. Look at `Git` -> `Local Changes` tab